modules/rds/ssm.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_ssm_parameter" "db_instance_address" {
+  name  = "/${var.name}/db_instance_address"
```

Comment:
also, can we remove the `db_instance_` from all the names?

Reply:
addres 😊

Reply:
done!